src/domain/request/RequestUrlParserService.js

```diff
@@ -0,0 +1,31 @@
+class RequestUrlParserService {
```

Comment:
```suggestion
class RequestUrlParser {
```
Same goes for the file name. I don't think `Service` suffix helps the name in any way